fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -52,23 +53,24 @@ object FAPI {
       configJson <- Response.Async.Right(faciaClient.config)
       frontJson <- Response.fromOption(configJson.fronts.get(frontId), NotFound(s"No front found for $frontId"))
       collectionIds = frontJson.collections
-      collectionsJsons <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsonOptions <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsons = collectionsJsonOptions.flatten
```

Comment:
To bring the conversation we had in here; this is what the defaults would be for that I mentioned. So we either change it back to an `Option` and then have the resolving of the default fields in the creation of `Collection`, or we keep this and change the zipping and use A default `CollectionJson`.